package.json

```diff
@@ -1,28 +1,63 @@
 {
-  "name": "fiscal",
-  "version": "1.0.0",
+  "name": "@winstoncl/fiscal",
```

Comment:
I think we need to nix `@winstoncl/`

Reply:
Sorry @MarkHerhold , just saw these comments, github didn't send me any notifications whatsoever!

yaa defo, I am just using that for now because I am hosting this package on github npm registry and using it, until these changes are merged (if they are) to the official one.